Makefile

```diff
@@ -28,6 +28,16 @@ VERSION_STRING=$(shell /usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionSt
 all: bootstrap
 	$(BUILD_TOOL) $(XCODEFLAGS) build
 
+sourcery:
+	sourcery --sources Tests --templates .sourcery/LinuxMain.stencil --output .sourcery
```

Comment:
Thanks for the tip! Although I need to shuffle things around to move the auto-generated Sourcery comments after the header anyways, so it doesn't really help in my case.